MuxExoPlayer/build.gradle

```diff
@@ -125,6 +125,6 @@ dependencies {
     r2_12_1_adsApi 'org.checkerframework:checker-qual:2.5.2'
     compileOnly 'com.google.ads.interactivemedia.v3:interactivemedia:3.9.0'
     compileOnly 'com.google.android.gms:play-services-ads:15.0.1'
-    compileOnly 'com.google.android.gms:play-services-ads-identifier:17.0.0'
+    compileOnly 'com.google.android.gms:play-services-ads-identifier:15.0.1'
```

Comment:
I think I'd had to bump to 17 at some point to make sure all of the flavors built and ran correctly, can you confirm that all of our build flavors, with and without ads, run correctly bumping this back? If so, we're okay with this or can stick to 17.

Reply:
I think all the tests are passing, so it is a good sign, it is possible that demo app crashes at some point, does it fail to build or does it crash when playing ads ?